examples/basic_usage.py

```diff
@@ -0,0 +1,57 @@
+"""
+Basic symbol search and ticker streaming example.
+"""
+import asyncio
+import time
+import traceback
+
+import ib_insync as ibi
+
+
+async def searchForContracts(ib, pattern):
+    """Search for multiple symbols concurrently using ``asyncio`` tasks.
+    """
+    descriptions = await ib.reqMatchingSymbolsAsync(pattern)
+    results = await asyncio.gather(
+        *[
+            ib.reqContractDetailsAsync(descr.contract)
+            for descr in descriptions
+        ]
+    )
+    return [cd.contract for cds in results for cd in cds]
+
+
+async def main():
+    """Entry point for our main ``asyncio`` task.
+    """
+    ib = ibi.IB()
+    with await ib.connectAsync(clientId=2):
```

Comment:
Just want to state that this is required with `asyncio.run()` otherwise you'll get a `RunTimeError` from `IB.__del__()` when it tries to disconnect the TCP protocol but the loop ain't running.